### PR TITLE
[ADD] l10n_in: merge l10n_in_upi l10n_in

### PR DESCRIPTION
**Pull request by kir-odoo**
merge l10n_in_upi into l10n_in

### CODE REVIEW

**Comment by jva-odoo** [comment]
@kir-odoo remove model file

**Review by jva-odoo — APPROVED** [review]

File: addons/l10n_in/models/res_company.py
Comment (jva-odoo) [inline addons/l10n_in/models/res_company.py:1]: alrady there is company.py

Comment (jva-odoo) [inline addons/l10n_in/models/res_company.py:1]: alrady there is company.py

**Review by jva-odoo — CHANGES_REQUESTED** [review]

**Comment by jco-odoo** [comment]
@jva-odoo @kir-odoo Is this still needed and if yes, -could this be updated so we can merge?  (it should have an upgrade script as well normally)

**Review by jva-odoo — APPROVED** [review]
@jco-odoo LGTM and ready for merged

File: addons/l10n_in/models/account_invoice.py [hunk 1]
```diff
@@ -122,3 +123,16 @@ def unlink(self):
         if logger_msg:
             _logger.info(logger_msg)
         return res
+
+    def _generate_qr_code(self, silent_errors=False):
+        self.ensure_one()
+        if self.company_id.country_code == 'IN':
+            payment_url = 'upi://pay?pa=%s&pn=%s&am=%s&tr=%s&tn=%s' % (
```
Comment (jco-odoo) [inline addons/l10n_in/models/account_invoice.py:130]: Unrelated with this, but it is still weird there is no url_encode/quote needed for this, but I think you said it was ok this way.

File: addons/l10n_in_upi/__manifest__.py [hunk 2]
```diff
@@ -1,19 +0,0 @@
-# -*- coding: utf-8 -*-
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
-{
-    'name': 'Indian - UPI',
-    'countries': ['in'],
-    'version': '1.0',
-    'description': """
-Invoice with UPI QR code
-=========================
-This module adds QR code in invoice report for UPI payment allowing to make payment via any UPI app.
```
Comment (jco-odoo) [inline addons/l10n_in_upi/__manifest__.py:10]: We could add something in the l10n_in manifest as well.

**Review by jco-odoo — APPROVED** [review]

**Pull request closed without merging**